packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/java/siw.md

```diff
@@ -0,0 +1,124 @@
+### 1: Start password recovery
+
+To begin the password recovery flow, the user must:
+
+1. Click the **Forgot Password?** link on the sign-in page.
+2. Enter their **Email or Username** in the box and click **Next**.
+3. Choose **Email** as the authenticator they want to use for password recovery and click **Submit**.
+
+Okta then sends the user an email that matches the Forgot Password template that you altered earlier, and the Sign-In Widget tells them to click the link in the email or to enter the OTP to continue.
+
+<div class="common-image-format">
+
+![Screenshot of email sent to user](/img/advanced-use-cases/custom-pwd-recovery-custom-email.png "Password Recovery Email")
```

Comment:
```suggestion
![Example of email sent to user](/img/advanced-use-cases/custom-pwd-recovery-custom-email.png "Password recovery email")
```
Maybe we don't need this image since it was already displayed for SDK steps?

Reply:
fixed. It's in the other guides so I'll leave this as is.